### PR TITLE
Rework specification to Status List 2021.

### PR DESCRIPTION
**Pull request by msporny**
This PR reworks the RevocationList2020 specification to support multiple types of status information; suspension and revocation.

### CODE REVIEW

**Review by kimdhamilton — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -272,50 +272,50 @@ <h3>RevocationList2020Status</h3>
...
   "type": ["VerifiableCredential"],
   "issuer": "did:example:12345",
-  "issued": "2020-04-05T14:27:42Z",
+  "issued": "2021-04-05T14:27:42Z",
   <span class="highlight">"credentialStatus": {
     "id": "https://dmv.example.gov/credentials/status/3#94567",
-    "type": "RevocationList2020Status",
-    "revocationListIndex": "94567",
-    "revocationListCredential": "https://example.com/credentials/status/3"
+    "type": "RevocationList2021",
```
Comment (dlongley) [inline index.html:316]: This is not the list itself, but information about a particular status *in* the list.

```suggestion
    "type": "RevocationList2021Status",
```

There will need to be minor fixes around this throughout the spec.

**Review by dlongley — CHANGES_REQUESTED** [review]

**Review by peacekeeper — APPROVED** [review]
Looks useful!

**Comment by peacekeeper** [comment]
Having multiple lists raises the question what to do if a status is set in multiple lists (e.g. "revoked" AND "suspended"). Also see my comment https://github.com/w3c-ccg/vc-status-rl-2020/issues/6#issuecomment-770686624.

Anyway, +1 to this.

File: index.html (hunk 1 above)
Comment (OR13) [reply index.html:316]: @tplooker please comment on this suggestion.

**Review by OR13 — APPROVED** [review]

**Review by csuwildcat — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -347,26 +425,26 @@ <h3>RevocationList2020Credential</h3>
...
               <td>id</td>
               <td>
-The <a>verifiable credential</a> that contains the revocation list MUST
+The <a>verifiable credential</a> that contains the status list MUST
 express an <code>id</code> property that matches the value specified in
-<code>revocationListCredential</code> for the corresponding
-<code>RevocationList2020Status</code>
-(see <a href="#revocationlist2020status"></a>).
+<code>statusListCredential</code> for the corresponding
+<code>RevocationList2021</code>
```
Comment (msporny) [inline index.html:431]:
```suggestion
<code>RevocationList2021Status</code>
```

File: index.html [hunk 3]
```diff
@@ -451,32 +529,32 @@ <h3>Validate Algorithm</h3>
...
-<a href="#revocationlist2020credential">RevocationList2020Credential</a>:
+<a href="#statuslist2021credential">StatusList2021Credential</a>:
       </p>
 
       <ol class="algorithm">
         <li>
 Let <strong>credentialToValidate</strong> be a <a>verifiable credentials</a>
 containing a <code>status</code> entry that is a
-<a href="#revocationlist2020status">RevocationList2020Status</a>.
+<a href="#revocationlist2021status">RevocationList2021</a>.
```
Comment (msporny) [inline index.html:539]:
```suggestion
<a href="#revocationlist2021status">RevocationList2021Status</a>.
```

File: index.html [hunk 4]
```diff
@@ -451,32 +529,32 @@ <h3>Validate Algorithm</h3>
...
 <code>encodedList</code> property of the
-<a href="#revocationlist2020credential">RevocationList2020Credential</a>.
+<a href="#statuslist2021credential">StatusList2021Credential</a>.
         </li>
         <li>
 Let <strong>credentialIndex</strong> be the value of the
-<code>revocationListIndex</code> property of the
-<a href="#revocationlist2020credential">RevocationList2020Status</a>.
+<code>statusListIndex</code> property of the
+<a href="#statuslist2021credential">RevocationList2021</a>.
```
Comment (msporny) [inline index.html:557]:
```suggestion
<a href="#statuslist2021credential">RevocationList2021Status</a>.
```

**Comment by msporny** [comment]
Normative, multiple reviews, changes requested and made, no objections, merging.

File: index.html (hunk 1 above)
Comment (tplooker) [reply index.html]: @OR13 not sure I follow, sorry only just saw this comment, I agree with the suggested change made by @dlongley here.

**Pull request merged**